Terminology change: s/encapsulated_key/sender_info (#16)
cose-wg/HPKE: draft-ietf-cose-hpke.md

  recipient use case with a two layer structure, the single 
  layer setup is more efficient.

> HPKE in "base" mode requires little information to be provided by

Reviewer: bikeshed but, according to RFC9180, Base mode might be better:

```suggestion
HPKE in Base mode requires little information to be provided by
```

   |         |                |            | defined by HPKE   |
   +---------+----------------+------------+-------------------+
~~~
{: #table-hpke-sender title="sender_info structure"}

  kem_id: This parameter is used to identify the KEM. The registry
          for KEM ids has been established with RFC 9180.

  kdf_id: This parameter contains the KDF identifier.
          The registry containing the KDF ids has been established
          with RFC 9180.

  aead_id: This parameter contains the AEAD identifier. The registry
           containing the AEAD ids has been established with RFC 9180.

>   enc: This parameter contains the output of the HPKE KEM.

Reviewer: I don't think this line needs to be modified.

```suggestion
  enc: This parameter contains the encapsulated key, which is output
       of the HPKE KEM.
```

## COSE Header Algorithm Parameters

-  Name: sender_info
-  Label: TBD2 (Assumed: -4)
-  Value type: sender_info
-  Value Registry: N/A

> -  Description: HPKE Sender Information structure for KEM-like algorithms

Reviewer: "KEM-like algorithms" is not necessary, but it would be better to limit it to Base mode.

```suggestion
-  Description: HPKE Sender Information structure for the Base mode.
```
